README.md

```diff
@@ -15,6 +16,15 @@ Generated Packages
 
 Generated OpenGL binding packages are available in the [go-gl/gl](https://github.com/go-gl/gl) repository.
 
+Overloads
+---------
+
+See subdirectory `xml/overload` for examples. The motivation here is to provide Go functions with different parameter signatures of existing OpenGL functions.
+
+For example, `glVertexAttribPointer(..., void *)` cannot be used with `gl.VertexAttribPointer(..., unsafe.Pointer)` when using arbitrary offset values. The `checkptr` safeguard will abort the program when doing so.
+Overloads allow to create an additional `gl.VertexAttribPointerWithOffset(..., uintptr)`, which calls the original function with appropriate casts.   
```

Comment:
tiny nit: "Overloads allow the creation an additional `gl.VertexAttribPointerWithOffset(..., uintptr)` function, which calls the original OpenGL function with appropriate casts."

Reply:
Done. Thank you for pointing this out. Explicit is better than implicit. :)